assets/scss/partials/footer.scss

```diff
@@ -1,20 +1,117 @@
+.main-footer {
+  display: grid;
+  grid-template-columns: 10% auto 10%;
+  grid-template-rows: repeat(1, 15rem);
```

Comment:
`grid-template-rows: repeat(1, 15rem)` is equivalent to `grid-template-rows: 15em`. Repeat is as a shorthand for when you want multiple rows or cols of the same width.